packages/aws-cdk/test/integ/cli/test-cdk-multiple-toolkit-stacks.sh

```diff
@@ -10,8 +10,8 @@ toolkit_stack_name_1="toolkit-stack-1-${RANDOM}"
 toolkit_stack_name_2="toolkit-stack-2-${RANDOM}"
 
 # deploy two toolkit stacks into the same environment (see #1416)
-cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_1}
-cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_2}
+cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_1} --tags Foo=Bar
+cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_2} --tags Foo=Bar
```

Comment:
nice! did you also want to verify that the tags were actually applied? I believe a `describe-stacks` call will contain that information in it.

Reply:
Thanks for the suggestion. I added it